### PR TITLE
W-12206167: Create a copy of loggingVariables to avoid race conditions

### PR DESCRIPTION
**Pull request by hyerans**

### CODE REVIEW

File: integration/src/test/java/org/mule/logging/TracingLoggingVariableTestCase.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.junit.Test;
+
+/**
+ * When setting logging variables with tracing module, race conditions can occur. This test will ensure that there are no race
+ * conditions when setting logging variables, with the fix for (W-12206167).
+ */
+public class TracingLoggingVariableTestCase extends MuleArtifactFunctionalTestCase {
```
Comment (fsgonz) [inline integration/src/test/java/org/mule/logging/TracingLoggingVariableTestCase.java:24]: add allure annotations.

**Review by fsgonz — CHANGES_REQUESTED** [review]

File: integration/src/test/java/org/mule/logging/TracingLoggingVariableTestCase.java (hunk 1 above)
Comment (hyerans) [reply integration/src/test/java/org/mule/logging/TracingLoggingVariableTestCase.java:26]: added @Issue allure annotation!

**Review by fsgonz — APPROVED** [review]

**Comment by hyerans** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Pull request closed without merging**